azure-governance-operations-automation/OMSAzureDashboards/OMSWebAppAzureMonitoringSolution.omsview

```diff
@@ -0,0 +1,295 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "defaultValue": ""
+        },
+        "resourcegroup": {
+            "type": "string",
+            "defaultValue": ""
+        },
+        "subscriptionId": {
+            "type": "string",
+            "defaultValue": ""
+        },
+        "workspace": {
+            "type": "string",
+            "defaultValue": ""
+        },
+        "workspaceapiversion": {
+            "type": "string",
+            "defaultValue": ""
+        }
+    },
+    "resources": [
+        {
+            "apiVersion": "[parameters('workspaceapiversion')]",
+            "name": "[parameters('workspace')]",
+            "type": "Microsoft.OperationalInsights/workspaces",
+            "location": "[parameters('location')]",
+            "id": "[Concat('/subscriptions/', parameters('subscriptionId'), '/resourceGroups/', parameters('resourcegroup'), '/providers/Microsoft.OperationalInsights/workspaces/', parameters('workspace'))]",
+            "resources": [
+                {
+                    "apiVersion": "[parameters('workspaceapiversion')]",
+                    "name": "37b624a0-31db-43f9-9f77-8effb87856b8",
+                    "type": "solutions",
+                    "location": "[parameters('location')]",
+                    "id": "[Concat('/subscriptions/', parameters('subscriptionId'), '/resourceGroups/', parameters('resourcegroup'), '/providers/Microsoft.OperationalInsights/workspaces/', parameters('workspace'),'/solutions/37b624a0-31db-43f9-9f77-8effb87856b8')]",
```

Comment:
the OMSView files are autogenerated by the Operations management views. These files are samples for customers to try. They are not deployed automatically (as there's no provision currently) with the ARM json.
however, i've fixed the lowercase , but this will get updated again until OMS fixes it on their side.